### ISSUE TITLE
How do I get started? (Please document.)

### ISSUE THREAD

**Issue by dchmelik** [airdcpp-web/airdcpp-webclient#244]
I compiled & installed airdcpp-webclient, set up the daemon & logged in, entered my WAN & LAN (‘bind’) IP addresses & three ports (TCP & UDP where needed,) opened those in router, tested them externally (they're open,) and airdcpp hashed my files, but when I connect to hubs, people say they can't get my list, and I can't get people's lists (at least passive users, which are the only I tried, when they said if I can read those, it works.) I'm new to DC++ and wish there was more documentation. What should I do to make this work? (and please consider documenting more, writing tutorials)

**Comment by maksis** [comment]
Have you read [DC++'s connectivity FAQ](http://dcplusplus.sourceforge.net/webhelp/faq_connection.html) or [additional help for manual connectivity config](http://dcplusplus.sourceforge.net/webhelp/settings_connectivity_manual.html)? Do they help at all?

If you have tested the ports and they are really open to internet, that should generally be enough. Did the IP you tested with match the one visible in hub's user list that is shown to you? Which port numbers did you use?

**Comment by dchmelik** [comment]
That documentation isn't for AirDC++ itself, (which could use more,) though originally helped, but I still have the problem. The IP address (static) matches. I used ports 51800 (TCP & UDP,) 51801 (TCP/TLS.)

**Comment by maksis** [comment]
What's the hubsoft being used?

**Comment by dchmelik** [comment]
What...?! I said (on airdcpp.net forum, but you hadn't seemed to have read all my post, saying stuff I obviously did if you had read, but directed me here) I'm new to DC++ and trying a client. I _never said_ I'm running a hub; I'm not. I tried different hubs and they all had this problem. One says PtokaX DC Hub 0.5.2.1.

**Comment by Kcchouette** [comment]
So if I understand:
 - you are connected on a hub
 - but the user hub can't see your filelist, and you can't see their filelist

My questions: 
 - Are you sure you have a share (on the menu > share)? Is it an empty share or does it contain folders/files?
 - Because it's DC++, do you have try to change the encoding filelist? (in the settings part of airdcpp-web)?

**Comment by dchmelik** [comment]
I said I shared files and they were hashed--yes (obviously) I have a share (otherwise would not have been hashed (a lot of files... took hours.) I don't understand your second question (ungrammatical.)

**Comment by maksis** [comment]
> One says PtokaX DC Hub 0.5.2.1.

Thanks, that's what I wanted to know. I'm not aware of any special issues caused by that hubsoft, unless it has been configured to block transfers. The hubsoft is written for a legacy protocol though, which may cause different types of issues. You may always connect to the dev/support hub at adcs://web-dev.airdcpp.net:1511 and ask for someone to try browsing your filelist.

Since you have presumably configured the application correctly and validated that the ports are really open, it's hard to come up with other suggestions.

**Comment by Kcchouette** [comment]
>You may always connect to the dev/support hub at adcs://web-dev.airdcpp.net:1511 and ask for someone to try browsing your filelist.

Good idea, it'll definitly check if your aridc++web instance is correctly configured

**Comment by maksis** [comment]
@dchmelik were you also able to download from other users in the dev hub?

**Comment by dchmelik** [comment]
I forgot to try that yet... I'll get to that later.

**Comment by dchmelik** [comment]
Okay, I was able to download from you.  So AirDC++ seems to be working fine, at least from my AirDC++ client to your AirDC++ hub.  As for the larger hub I retried that may still have the version above, it just lists me as passive mode (which did not happen on your hub.)  I didn't change anything or do anything in between, and thought I made that clear, but someone running the other hub doesn't believe me. My ports also didn't randomize in between, as I've set them specifically.  I guess we may as well close this issue and I'll talk to that hub's administrator.
